js/src/jit/BaselineCacheIRCompiler.cpp

```diff
@@ -2499,54 +2503,67 @@ static bool AddToFoldedStub(JSContext* cx, const CacheIRWriter& writer,
   return true;
 }
 
-ICAttachResult js::jit::AttachBaselineCacheIRStub(
-    JSContext* cx, const CacheIRWriter& writer, CacheKind kind,
-    JSScript* outerScript, ICScript* icScript, ICFallbackStub* stub,
-    const char* name) {
-  // We shouldn't GC or report OOM (or any other exception) here.
-  AutoAssertNoPendingException aanpe(cx);
-  JS::AutoCheckCannotGC nogc;
+#ifdef ENABLE_JS_AOT_ICS
+void DumpNonAOTICStubAndQuit(CacheKind kind, const CacheIRWriter& writer) {
+  // Generate a filename based on the current time and PID.
```

Comment:
I think this comment is out-of-date as it looks like the filename is now a random uint64 appended to "IC-"

Reply:
Updated!